hnn/paramrw.py

```diff
@@ -130,10 +133,27 @@ def countEvokedInputs(params):
                     ndist += 1
     return nprox, ndist
 
-# check if using any evoked inputs
+
+def _get_ordered_param_inputs(params):
+    times_dict = {}
+    input_list = []
+
+    # first pass through all params to get mu and sigma for each
+    for k in params.keys():
+        input_mu = re.match('^t_ev(prox|dist)_([0-9]+)', k)
+        if input_mu is not None:
+            id_str = 'ev' + input_mu.group(1) + '_' + input_mu.group(2)
+            times_dict[id_str] = float(params[k])
+
+    sorted_inputs = dict(sorted(times_dict.items(), key=lambda item: item[1]))
+    for input_str in sorted_inputs.keys():
+        input_list.append(input_str)
```

Comment:
Not sure if this is a realistic failure point, but perhaps a test to ensure the numbering is sequential (t_evprox_1 -> t_exprox5) for example. Something like `assert np.all(np.diff(ev_index_list) == 1))`

Reply:
Yes, very reasonable... that is the check that would have alerted me to the issue addressed by commit 35352a438f. I just won't have the time to implement CI checks for HNN GUI.